platform/src/main/java/org/stellar/anchor/platform/service/PaymentOperationToEventListener.java

```diff
@@ -113,12 +116,17 @@ public void onReceived(ObservedPayment payment) {
       txn.setStellarTransactionId(payment.getTransactionHash());
       try {
         transactionStore.save(txn);
+        Metrics.counter("sep31.transaction",
+                "status", SepTransactionStatus.PENDING_RECEIVER.toString()).increment();
       } catch (SepException ex) {
         Log.errorEx(ex);
       }
     }
     // send to the event queue
     sendToQueue(event);
+    Metrics.counter(
+            "payment.received", "toAddress", payment.getTo(), "asset", payment.getAssetName())
+        .increment();
```

Comment:
👀  Getting one counter per receiver can be a hassle to visualize.

‼️  Also, some metrics tools charge per different metric, so a client with high throughput could end up spending a ton on metrics services if we ship this kind of metric. This is something to be logged rather than a metric.

Reply:
Not necessarily, because we may use custodial solutions like Circle that can provide a new receiving address per Transaction.